.dev/CRAN_Release.cmd

```diff
@@ -39,6 +39,9 @@ grep --exclude="./src/openmp-utils.c" omp_get_max_threads ./src/*
 # Ensure all #pragama omp parallel directives include a num_threads() clause
 grep "pragma omp parallel" ./src/*.c | grep -v getDTthreads
 
+# Update documented list of places where openMP parallelism is used: c.f. ?openmp
+grep -Elr "#pragma omp (?:for|parallel)" src | sort
```

Comment:
@MichaelChirico we also use `_Pragma("omp parallel for num_threads(getDTthreads())")` so probably better to remove leading `#`

Reply:
Nice spot. Changed here: 5730cf8f96c3d9febb4b144976965f29382741ae. It caught one more file (freadR.c, as well as fread.c, making 17 files total) so nothing else to update.